packages/thirdweb/src/rpc/watchBlockNumber.ts

```diff
@@ -49,8 +49,6 @@ function createBlockNumberPoller(
   async function poll() {
     // stop polling if there are no more subscriptions
     if (!isActive) {
-      lastBlockNumber = undefined;
-      lastBlockAt = undefined;
```

Comment:
just checking we don't need this anymore? I know @jnsdls added this to fix a bug when subscribing after a few blocks that would re-play a bunch of events.

that said there is a unit test for it now so I think we should be good!

Reply:
yeah if the unit tests are happy then I'm happy!